connect/runtime/src/test/java/org/apache/kafka/connect/integration/ExampleConnectIntegrationTest.java

```diff
@@ -56,7 +56,9 @@
     private static final int NUM_TOPIC_PARTITIONS = 3;
     private static final int CONSUME_MAX_DURATION_MS = 5000;
     private static final int CONNECTOR_SETUP_DURATION_MS = 15000;
+    private static final int WORKER_SETUP_DURATION_MS = 20_000;
```

Comment:
Please skip this file and review: https://github.com/apache/kafka/pull/6342 instead. Thanks!